### PR TITLE
[CLD-3936] Improve pipelines and makefile for Elrond image release

### PR DESCRIPTION
**Pull request by stylianosrigas**
#### Summary
- Improve pipelines and makefile for Elrond image release

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,34 +61,19 @@ binaries: ## Build binaries of elrond
...
 .PHONY: build
-build:
+build: ## Build the elrond
 	@echo Building Elrond
-	GOOS=linux CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/main ./cmd/$(APP)
-
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 $(GO) build -ldflags '$(LDFLAGS)' -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/elrond  ./cmd/elrond
 
-.PHONY: build-image
-build-image:  ## Build the docker image for Elrond
```
Comment (stafot) [inline Makefile:71]: Is there reasoning behing removing arm builds?

Comment (stylianosrigas) [reply Makefile:71]: @stafot Can you have another look. Was still work in progress.

File: Makefile [hunk 2]
```diff
@@ -61,34 +61,19 @@ binaries: ## Build binaries of elrond
...
+build-image:  ## Build the docker image for elrond
 	@echo Building Elrond Docker Image
-	echo $(DOCKER_PASSWORD) | docker login --username $(DOCKER_USERNAME) --password-stdin
-	docker buildx build \
-	--platform linux/arm64,linux/amd64 \
+	docker build \
```
Comment (stafot) [inline Makefile]: Here we changed to `docker build` instead of `docker buildx` which supports multi-arch builds. @stylianosrigas

File: Makefile [hunk 3]
```diff
@@ -61,34 +61,19 @@ binaries: ## Build binaries of elrond
...
 	GOOS=darwin GOARCH=arm64 CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/elrond-darwin-arm64  ./cmd/$(APP)
 
+
 .PHONY: build
-build:
+build: ## Build the elrond
 	@echo Building Elrond
-	GOOS=linux CGO_ENABLED=0 $(GO) build -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/main ./cmd/$(APP)
-
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 $(GO) build -ldflags '$(LDFLAGS)' -gcflags all=-trimpath=$(PWD) -asmflags all=-trimpath=$(PWD) -a -installsuffix cgo -o build/_output/bin/elrond  ./cmd/elrond
```
Comment (stafot) [inline Makefile:68]: @stylianosrigas we might want to add here also also `arm64` build.

**Review by stafot — APPROVED** [review]

**Pull request merged**